src/main.rs

```diff
@@ -40,17 +41,16 @@ fn main() -> Result<(), Box<dyn Error>> {
         };
     }
 
-    let initial_pose_pub = initial_pose::InitialPosePub::new(
-        &conf.robot_frame,
-        static_frame.clone(),
-    );
+    let initial_pose_pub =
+        initial_pose::InitialPosePub::new(&conf.robot_frame, static_frame.clone());
```

Comment:
did you un rust format here?

Reply:
actually this was done by rust format i had a lb before

Reply:
I think it got lost during a rebase previously and fixed now. All good